python/ray/data/read_api.py

```diff
@@ -3277,6 +3277,7 @@ def read_clickhouse(
     table: str,
     dsn: str,
     columns: Optional[List[str]] = None,
+    filters: Optional[str] = None,
```

Comment:
nit: Let's make it singular, since we're passing one filter (potentially consisting of a few predicates)